package.json

```diff
@@ -121,5 +121,13 @@
   },
   "dependencies": {
     "uuid": "^9.0.0"
+  },
+  "pnpm": {
+    "overrides": {
+      "minimist": "^1.2.8",
+      "semver": "^7.5.4",
+      "socks": "^2.7.3",
+      "string_decoder": "^1.3.0"
```

Comment:
It looks like you are adding these here but I see from the change in the lock file that some were already being overridden there -- do you know what causes that? How did there get to be overrides in the lock file that weren't listed in package.json?

Reply:
When migrating from `yarn` to `pnpm` the resolutions field is useful to override packages as `pnpm import` commands generates `pnpm-lock.yaml` file. Now when I added `"pnpm": {"overrides": }` field in `package.json` then `pnpm.overrides` takes precedence over `resolutions` field as mentioned in the [pnpm resolutions](https://pnpm.io/next/package_json#resolutions) documentation. To resolve this, I've updated `resolutions` to `pnpm.overrides`.